src/pages/projects.js

```diff
@@ -508,6 +508,33 @@ const ProjectDescriptions = () => (
         </div>
       </div>
     </div>
+
+    <div className="project-box">
+      <a name="tracee" />
+      <div className="project-major-title">Tracee</div>
+      <div className="project-minor-title">Dynamic Security Event Tracing</div>
```

Comment:
As a native English speaker I'm going to respectfully disagree here - "security-event tracing" looks wrong to me. I think it's because "security" is an adjective describing the event(s), but not the tracing of those events. 

My first choice here is "Dynamic Security Event Tracing"; if we can't have that, I'd say "Dynamic Tracing for Security Events"

Reply:
> I think it's because "security" is an adjective describing the event(s), but not the tracing of those events.

As far as I understand, this is exactly why we usually use a hyphen (ex. video-game industry, credit-card debt, etc.). Reading more about this, I see at least two reasons why we might want to *not* use a hyphen here: 1) some websites recommend against the use of hyphens in titles, and 2) the rules for noun+noun compound adjectives seem less clear and more prone to regional preferences (US/UK).

In any case, if this sounds weird to you, let's drop the hyphen. This, as well as the comma mentioned below, is just a nitpick :slightly_smiling_face: